dash/dash.py

```diff
@@ -1302,46 +1308,40 @@ def enable_dev_tools(self,
         :type dev_tools_silence_routes_logging: bool
         :return: debug
         """
-        env = _configs.env_configs()
-        debug = debug or _configs.get_config('debug', None, env, debug,
-                                             is_bool=True)
-
-        self._dev_tools.dev_tools_ui = debug
+        debug = debug or get_combined_config('debug', None, debug)
 
-        self._dev_tools['serve_dev_bundles'] = _configs.get_config(
-            'serve_dev_bundles', dev_tools_serve_dev_bundles, env,
-            default=debug,
-            is_bool=True
+        self._dev_tools['dev_tools_ui'] = get_combined_config(
+            'dev_tools_ui', dev_tools_ui, default=True
```

Comment:
I think these should be `default=debug` like the rest of the properties. That way, it won't accidently appear on production.

For users working in development, we popularize `debug=True` by including it in all of our docs:
```python
if __name__ == '__main__':
     app.run_server(debug=True)
```

So, in effect, most users will get all of these features turned on by default since `debug=True`. However, since `run_server` isn't used in production, it'll default to `False`.